services/static-webserver/client/source/class/osparc/info/ServiceUtils.js

```diff
@@ -295,23 +295,19 @@ qx.Class.define("osparc.info.ServiceUtils", {
     },
 
     patchServiceData: function(serviceData, fieldKey, value) {
-      return new Promise((resolve, reject) => {
-        const patchData = {};
-        patchData[fieldKey] = value;
-        const params = {
-          url: osparc.data.Resources.getServiceUrl(
-            serviceData["key"],
-            serviceData["version"]
-          ),
-          data: patchData
-        };
-        osparc.data.Resources.fetch("services", "patch", params)
-          .then(() => {
-            serviceData[fieldKey] = value;
-            resolve();
-          })
-          .catch(err => reject(err));
-      });
+      const patchData = {};
+      patchData[fieldKey] = value;
+      const params = {
+        url: osparc.data.Resources.getServiceUrl(
+          serviceData["key"],
+          serviceData["version"]
+        ),
+        data: patchData
+      };
+      osparc.data.Resources.fetch("services", "patch", params)
```

Comment:
```suggestion
      return osparc.data.Resources.fetch("services", "patch", params)
```

Reply:
like that you can also
``
ServiceUtils.patchServiceData(...).then(...)...
``

Reply:
Forgot to return it, thanks